modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SwiftCodegen.java

```diff
@@ -115,10 +117,10 @@ public SwiftCodegen() {
         typeMapping.put("string", "String");
         typeMapping.put("char", "Character");
         typeMapping.put("short", "Int");
```

Comment:
You're right, looks like it won't work anyways. 
I agree keeping it untouched.
I think we can say the Int to Int16 fix is noncrucial compared to the Int to Int32 fix.